lang/en/docs/workflows/data/units.md

```diff
@@ -44,6 +45,36 @@ For a description of unit input templating, the reader is referred to [this sect
 {!example/workflow/unit/io.json!}
 ```
 
+### API (Input Section Only)
```

Comment:
We don't need to specify "Input Section Only"

Reply:
Removed